apps/api-extractor-model/src/items/ApiDeclaredItem.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.s
 
-import { DeclarationReference } from '@microsoft/tsdoc/lib/beta/DeclarationReference';
+import { DeclarationReference } from '@microsoft/tsdoc/lib-commonjs/beta/DeclarationReference';
```

Comment:
Should this be exposed from the root of the TSDoc package?

It'd be cool if we were able to somehow rewrite these kinds of deep import during compilation to match the output format (i.e. - for the commonJS output, this would be `@microsoft/tsdoc/lib-commonjs/beta/DeclarationReference`, for the ESNext output, this would be `@microsoft/tsdoc/lib-esnext/beta/DeclarationReference`)

Reply:
> Should this be exposed from the root of the TSDoc package?

Yes.  I just haven't had time yet to  integrate it into the real TSDoc API. Until then it's an unofficial API  and not a big deal if it's awkward to reference.

However, this path-based import is a legitimate situation, and Rush Stack / Heft do need a way to handle it.  @iclanton if there isn't a simple straightforward solution (e.g. emit .d.ts files in `lib-commonjs`?) then we should create a GitHub issue to figure out the design.